### DISCUSSION TITLE
What version of download should i use?

### DISCUSSION THREAD

**Discussion by LightslicerGP** [bridge-core/bridge.#404] [Q&A]
I see that there are mac and Linux versions but I cant find the right .exe version for my Windows 10 device, which should I use, and what does each mean?

**Comment by Chikorita-Lover** [comment]
There's only one EXE download available

**Comment by LightslicerGP** [comment]
> There's only one EXE download available

That says web installation, and I assume it would be some version that uses the website

**Comment by solvedDev** [comment]
It has "web" inside of it because it is a minimal installer that fetches the actual app binaries from the web depending on whether you use a 32bit or 64bit processor
